### PR TITLE
Add First Version of Repolinter Action

### PR DESCRIPTION
**Pull request by IsaacMilarky**
## Problem

Repolinter is not currently run automatically as a check GH action. 

## Solution

Use [repolinter-action](https://github.com/newrelic/repolinter-action) to run repolinter as an action. This action is using the tier 3 repolinter.json that was created. This is because if I remember correctly Metrics is a tier 3 project.

The action will create an issue on the project to give remediation steps to follow tier 3 project requirements.

### CODE REVIEW

**Comment by IsaacMilarky** [comment]
There seems to be some problems with the format of the `repolinter.json` file. I will look into this but shouldn't merge in the meantime. 

```
 Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Warning: The `set-output` command is deprecated and will be disabled soon. Please upgrade to using Environment Files. For more information see: https://github.blog/changelog/2022-10-11-github-actions-deprecating-save-state-and-set-output-commands/
Error: A fatal error was thrown.
Error: Error: Configuration validation failed with error Configuration validation failed with errors: 
	configuration.rules['maintainers-file-exists'] should have required property 'rule'
    at Object.rejected (/repolinter-action/webpack:/repolinter-action/lib/getConfig.js:109:1)
    at Generator.next (<anonymous>)
    at fulfilled (/repolinter-action/webpack:/repolinter-action/lib/getConfig.js:25:1)
```

**Comment by IsaacMilarky** [comment]
Ready to merge now.

File: repolinter.json [hunk 1]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}CONTRIBUTING.md"
+					],
+					"content": "Building the Project",
+					"flags": "i"
+				}
+			}
+		},
+		"contributing-contains-workflow-and-branching": {
+			"level": "warning",
```
Comment (natalialuzuriaga) [inline repolinter.json:503]: level should be "error"

File: repolinter.json [hunk 2]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}CONTRIBUTING.md"
+					],
+					"content": "Testing Conventions",
+					"flags": "i"
+				}
+			}
+		},
+		"contributing-contains-coding-style-and-linters": {
+			"level": "warning",
```
Comment (natalialuzuriaga) [inline repolinter.json:529]: level should be "error"

File: repolinter.json [hunk 3]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}CONTRIBUTING.md"
+					],
+					"content": "Reviewing Pull Requests",
+					"flags": "i"
+				}
+			}
+		},
+		"contributing-contains-shipping-releases": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:581]: level should be "warning"

File: repolinter.json [hunk 4]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}CODEOWNERS.md"
+					],
+					"content": "documentation | frontend | backend | Repo Domains",
+					"flags": "i"
+				}
+			}
+		},
+		"community-guidelines-contains-principles": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:750]: level should be "error"

File: repolinter.json [hunk 5]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}COMMUNITY_GUIDELINES.md"
+					],
+					"content": "Principles",
+					"flags": "i"
+				}
+			}
+		},
+		"community-guidelines-contains-community-guidelines": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:763]: level should be "error"

File: repolinter.json [hunk 6]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}COMMUNITY_GUIDELINES.md"
+					],
+					"content": "Community Guidelines",
+					"flags": "i"
+				}
+			}
+		},
+		"community-guidelines-contains-acknowledgements": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:776]: level should be "error"

File: repolinter.json [hunk 7]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}COMMUNITY_GUIDELINES.md"
+					],
+					"content": "Acknowledgements",
+					"flags": "i"
+				}
+			}
+		},
+		"code-of-conduct-contains-contributor-code-of-conduct": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:789]: level should be "error"

File: repolinter.json [hunk 8]
```diff
@@ -1,367 +1,815 @@
...
+					"globsAll": [
+						"{docs/,.github/,}CODE_OF_CONDUCT.md"
+					],
+					"content": "Contributor Code of Conduct",
+					"flags": "i"
+				}
+			}
+		},
+		"code-of-conduct-contains-acknowledgements": {
+			"level": "off",
```
Comment (natalialuzuriaga) [inline repolinter.json:802]: level should be "error"

**Review by natalialuzuriaga — CHANGES_REQUESTED** [review]
Since we are unable to extend from the Tier 2 repolinter.json file, I noted a couple of checks whose levels need to be changed to match Tier 3 requirements

**Comment by IsaacMilarky** [comment]
> Since we are unable to extend from the Tier 2 repolinter.json file, I noted a couple of checks whose levels need to be changed to match Tier 3 requirements

Oh my bad I forgot to merge the Tier 3 changes into the JSON. Will fix

**Review by natalialuzuriaga — APPROVED** [review]
LGTM! ✅

**Pull request merged**